### ISSUE TITLE
Relative and absolute tolerances for integrators?

### ISSUE THREAD

**Issue by chr218** [hgrecco/numbakit-ode#22]
It would be nice to have the option to set relative and absolute tolerances for the integrators, like in "scipy.integrate.solve_ivp".

Thanks!

**Comment by maurosilber** [comment]
It is already possible to do that on variable step methods, but it's not yet documented. For example:

```python
import nbkode

def func(t, y):
    return -y

>>> nbkode.RungeKutta45(func, 0, 1).run(1)
(array([1]), array([[0.37535081]]))

>>> nbkode.RungeKutta45(func, 0, 1, rtol=1e-12, atol=1e-12).run(1)
(array([1]), array([[0.36789539]]))
```

**Comment by chr218** [comment]
Which version is this? I have attempted to reproduce your code, but got the
following error below:


























*import nbkodedef func(t, y):    return -y    nbkode.RungeKutta45(func, 0,
1).run(1)C:\Users\chr218\Anaconda3\lib\site-packages\nbkode\core.py:532:
NumbaExperimentalFeatureWarning: First-class function type feature is
experimental  if not step(t_bound, rhs, cache,
*args):C:\Users\chr218\Anaconda3\lib\site-packages\nbkode\core.py:532:
NumbaExperimentalFeatureWarning: First-class function type feature is
experimental  if not step(t_bound, rhs, cache, *args):Out[10]: (array([1]),
array([[0.37535081]]))nbkode.RungeKutta45(func, 0, 1, rtol=1e-12,
atol=1e-12).run(1)Traceback (most recent call last):  File
"<ipython-input-11-a5a642af71f1>", line 1, in <module>
nbkode.RungeKutta45(func, 0, 1, rtol=1e-12, atol=1e-12).run(1)  File
"C:\Users\chr218\Anaconda3\lib\site-packages\nbkode\core.py", line 577, in
__init__    super().__init__(*args, **kwargs)  File
"C:\Users\chr218\Anaconda3\lib\site-packages\nbkode\runge_kutta\core.py",
line 26, in __init__    super().__init__(*args, **kwargs)TypeError:
__init__() got an unexpected keyword argument 'rtol'*

**Comment by maurosilber** [comment]
It was fixed in 1ab77ab8fff1616b606b5ff19b55ccd924cc332a, but there hasn't been a release to PyPI since then. Sorry. You would have to install from GitHub: `pip install git+https://github.com/hgrecco/numbakit-ode`
